src/DssSpell.sol

```diff
@@ -60,222 +58,192 @@ contract DssSpellAction is DssAction {
 
-        // Byteron - 8333 DAI - 0xc2982e72D060cab2387Dba96b846acb8c96EfF66
-        DssExecLib.sendPaymentFromSurplusBuffer(BYTERON, 8_333);
+            // Note: Migrate all but 200 million USDC reserves from PSM-USDC-A to LITE-PSM-USDC-A
+            srcKeep:    200 * MILLION * WAD
+        });
 
-        // JuliaChang - 8333 DAI - 0x252abAEe2F4f4b8D39E5F12b163eDFb7fac7AED7
-        DssExecLib.sendPaymentFromSurplusBuffer(JULIACHANG, 8_333);
+        // Note: LitePSM migration was extracted into a library,
+        //       and implemented as part of the LitePSM module.
+        DssLitePsmMigrationPhase2.migrate(dss, cfg);
 
-        // Rocky - 7500 DAI - 0xC31637BDA32a0811E39456A59022D2C386cb2C85
-        DssExecLib.sendPaymentFromSurplusBuffer(ROCKY, 7_500);
+        // ----- GSM Delay Update -----
+        // Forum: https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839
+        // Poll: https://vote.makerdao.com/polling/QmU7XJ6X
 
-        // PBG - 6667 DAI - 0x8D4df847dB7FfE0B46AF084fE031F7691C6478c2
-        DssExecLib.sendPaymentFromSurplusBuffer(PBG, 6_667);
+        // Increase the GSM Pause Delay by 14h, from 16h to 30h
+        PauseLike(MCD_PAUSE).setDelay(30 hours);
 
-        // ---------- RWA001-A Stability Fee Increase ----------
-        // Forum: https://forum.makerdao.com/t/rwa-001-6s-capital-update-and-stability-fee-proposal/24624
+        // ----- Update LitePSM Keeper Network Job -----
+        // Forum: https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839
+        // Poll: https://vote.makerdao.com/polling/QmU7XJ6X
 
-        // Increase the RWA001-A Stability Fee by 6 percentage points from 3% to 9%
-        DssExecLib.setIlkStabilityFee("RWA001-A", NINE_PCT_RATE, /* doDrip = */ true);
+        // Remove the old LitePSMJob (0x689cE517a4DfCf0C5eC466F2757D324fc292C8Be) from the CronSequencer
+        DssCronSequencerLike(CRON_SEQUENCER).removeJob(CRON_LITE_PSM_JOB);
 
-        // Note: Bump Oracle price to account for new SF
-        // Note: the formula is `Debt ceiling * [ (1 + RWA stability fee ) ^ (minimum deal duration in years) ] * liquidation ratio`
-        // Since RWA001-A Termination Date is `July 29, 2025`, and spell execution time is `2024-08-12`, the distance is `356` days
-        // bc -l <<< 'scale=18; 15000000 * e(l(1.09) * (352/365)) * 1.00' | cast --to-wei
-        RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).bump(
-            "RWA001-A",
-            16_299_893_185222593795000000
-        );
+        // Add the new LitePSMJob (0x0c86162ba3e507592fc8282b07cf18c7f902c401) to the Cron Sequencer
+        DssCronSequencerLike(CRON_SEQUENCER).addJob(CRON_LITE_PSM_JOB_NEW);
 
-        // Note: Update collateral price to propagate the changes
-        DssExecLib.updateCollateralPrice("RWA001-A");
+        // fill: Set the rushThreshold to 20 million DAI
+        // Note: Value already set at target contract, cannot be changed
```

Comment:
Nit: for the external reader "cannot be changed" might sound like you can't implement requested instruction, which is not the actual reason why it's not taken here
```suggestion
        // Note: The value is already set in CRON_LITE_PSM_JOB_NEW
```

Reply:
fixed in [ce88b5e](https://github.com/makerdao/spells-mainnet/pull/422/commits/ce88b5ee5cfc86ea65989cfdb00948334dec4fde)